template/src/overrideFormioStyles.less

```diff
@@ -131,3 +131,15 @@
 main:focus {
   outline: none;
 }
+
+.margin-bottom-default {
+  margin-bottom: 1rem;
+}
+
+.margin-bottom-large {
+  margin-bottom: 4rem;
+}
+
+.margin-left-default {
+  margin-left: 1rem;
+}
```

Comment:
Mangler newline, ikke noe krise men gjør at det ser rart ut når man bruker en del tekst baserte tools

Reply:
Høres ut som en mangel i prettier/lint-configen. Bør legge til regel i Husky for å kjøre på commit, slik som for JS.
Tas i etterkant.